bigc/resources/pricing.py

```diff
@@ -0,0 +1,18 @@
+from collections.abc import Iterator
+
+from bigc._client import BigCommerceV3APIClient
+
+
+class BigCommercePricingAPI:
+    def __init__(self, store_hash: str, access_token: str):
+        self._v3_client = BigCommerceV3APIClient(store_hash, access_token)
+
+    def products(self, customer_group_id: int, items: list[dict], channel_id: int = 1,
+                 currency_code: str = 'USD') -> Iterator[dict]:
```

Comment:
Let's put `items` first, so we can require that all the other arguments must be keyword arguments for clarity.
```suggestion
    def products(self, items: list[dict], *, customer_group_id: int, channel_id: int = 1,
                 currency_code: str = 'USD') -> Iterator[dict]:
```
Also, I'm not sure why we've elected to provide a default for `channel_id` but not for `customer_group_id` here. I think `0` is the "no group" ID.

Reply:
The docs don't specify any default value so I didn't provide one.